base-template/about-base-template.md

```diff
@@ -1,8 +1,21 @@
 # About Base Template
 
+{Template author tip:
+
+* This about-base-template captures the structure and common elements that all downstream about-xxx-templates should follow.
+* It provides guidance for both template authors as well as document authors.
+* “Template author and template reviewer tips” should be removed from the final about-{doctype}-template.md. Other tips should remain.
+* Tweak other tips to align with the doctype you are writing for.
+
+}
+
+This document provides guidance related to the authoring of documents based on the {doctype}-template. It focuses on the “how”, “why” and where applicable, references background theory to justify the approaches recommended.
```

Comment:
```suggestion
This document provides guidance related to the authoring of documents based on the {doctype}-template. It focuses on the “how” and “why”; and where applicable, references background theory to justify the approaches recommended.
```

Reply:
A bunch or rewording, and this section has been removed. I think I've fixed whatever quotes we had before.